base/src/main/java/uk/ac/ebi/atlas/model/Publication.java

```diff
@@ -1,11 +1,24 @@
 package uk.ac.ebi.atlas.model;
 
+import org.codehaus.jackson.annotate.JsonIgnoreProperties;
+import org.codehaus.jackson.annotate.JsonProperty;
```

Comment:
I like Jackson! I can see ourselves gradually moving away from Gson in the future.